reservations/views.py

```diff
@@ -1,3 +1,16 @@
 from django.shortcuts import render
+from django.views.generic import ListView, DetailView
 
-# Create your views here.
+from .models import Reservation
+
+
+class ReservationList(ListView):
+    """Lists all the reservations associated with a room or a user"""
+    model = Reservation
+    template_name = 'reservations/reservation_list.html'
```

Comment:
You're right that we will want to get lists of reservations per room and per user, but this view isn't doing that. This is ALL rooms (think of it like an admin view of everything that's going on in the system). We should change the docstring so it gives accurate info